src/views/CreateType.vue

```diff
@@ -43,11 +43,11 @@
         />
         <Select
           :value.sync="editingAttribute.dataType"
-          name="Data Type"
+          name="Type"
```

Comment:
if I try to  add a new attribute nothing happens.  --> please fix this
(klicking "add Attribute" does not open the Thing where you can select Type and Name of the new Attribute)

Reply:
Thanks for spotting that, I will look into it.